server/utils/getJWTSecret.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-console */
+const getJWTSecret = () => {
+  try {
+    if (process.env.JWT_SECRET === undefined)
```

Comment:
if (!process.env.JWT_SECRET === undefined)

Reply:
You mean `if (!process.env.JWT_SECRET)` ?
Switched to that in `jwt-authentication-login-page`.